client/src/components/Navbar/Navbar.tsx

```diff
@@ -77,16 +83,20 @@ const menuItems = [
 ];
 
 const MenuItem: React.FC<{
-  resource: string;
+  resource?: string;
   item: string | JSX.Element;
-}> = ({ resource, item }) => {
+  index: number;
+}> = ({ resource, item, index }) => {
   const classes = useStyles();
 
   return (
-    <Grid key={resource} sx={{ textAlign: 'center' }} xs={2} justifySelf="flex-end" item>
-      <NavLink className={classes.navbarItem} to={resource}>
-        {item}
-      </NavLink>
+    <Grid key={resource || index} sx={{ textAlign: 'center' }} xs={2} justifySelf="flex-end" item>
```

Comment:
Since the number of `MenuItem`s will dynamically change based on the user type, usage of indexes as keys is not recommended and considered an anti-pattern

Reply:
Sorry, I just implemented this change because the Notifications button does not have a resource or "href" that it travels to. Thank you for the heads up :)